TFNetworkRecLayer.py

```diff
@@ -4670,10 +4670,13 @@ def _weights_time_axis_to_reduce(cls, weights, base):
     # SoftmaxOverSpatialLayer by default uses time_dim_axis.
     # We also should maybe check that this matches the base time dim axis.
     dyn_axes = weights.get_dynamic_axes()
+
     # for static time-dim
-    if weights.time_dim_axis not in dyn_axes:
+    if not dyn_axes or weights.time_dim_axis not in dyn_axes:
+      assert weights.time_dim_axis is not None and weights.time_dim_axis is not NotSpecified
```

Comment:
It never can be `NotSpecified`. That check does not make sense.
Also, I don't quite understand how this fixes a problem.

Reply:
This seems to be a merge error, as the check for static time axis did not exist when I made the change a long time ago (but never did a commit for it before), so the commit can be removed

Reply:
Ok, then remove it, and update the PR.

Reply:
And resolve this when done.

Reply:
I guess this is resolved?